subprojects/docs/src/docs/userguide/commandLine.adoc

```diff
@@ -0,0 +1,142 @@
+// Copyright 2017 the original author or authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[appendix]
+[[gradle_command_line]]
+== Gradle Command Line
+
+The *`gradle`* command has the following usage:
+
+----
+gradle [option...] [task...]
+----
```

Comment:
This looks different.
Before: 
![image](https://user-images.githubusercontent.com/423186/27594202-0c1c1d7c-5b0e-11e7-8ead-21d814637ac0.png)
After: 
![image](https://user-images.githubusercontent.com/423186/27594234-1d30b0be-5b0e-11e7-8973-483378fd8e2b.png)